temporal/client.go

```diff
@@ -34,7 +34,8 @@ type Configuration struct {
 
 var Module = fx.Module(
 	"temporal",
-	fx.Provide(ArmoryTemporalProvider),
+	fx.Provide(ClientProvider),
+	fx.Provide(WorkerProviderProvider),
```

Comment:
I am not a huge fan of "ProviderProvider", but maybe there isn't a more clear name to use? :-D